apps/web-giddh/src/assets/css/style-1.scss

```
@@ -2749,3 +2749,68 @@ form#groupDetailForm {
         max-width: 280px;
     }
 }
+.mat-dialog-container {
+    padding: 0px !important;
+}
+.mat-dialog-title {
+    margin-bottom: 0px !important;
+}
+mat-dialog-container .modal-header {
+    padding: 11px !important;
+}
+.mat-radio-button {
+    margin-right: 16px;
+}
+.mat-grid-tile-content {
+    justify-content: left !important;
+    display: block !important;
+}
+.ledg-head mat-grid-list mat-grid-tile .mat-grid-tile-content {
+    padding: 10px 15px !important;
+}
+.ledger-head mat-grid-list mat-grid-tile .mat-grid-tile-content {
+    padding-top: 5px !important;
+    padding-bottom: 10px !important;
+    padding-left: 16px !important;
+    font-size: 14px;
+    font-family: "opensans-semibold";
+}
+#ledger-transaction-left-list mat-grid-tile .mat-grid-tile-content, #ledger-transaction-right-list mat-grid-tile .mat-grid-tile-content {
+    padding: 1px !important;
+    padding-right: 10px !important;
+}
+#eledgerwrap .alert .mat-close {
+    position: absolute;
+    top: 10px;
+    right: 0px;
+    opacity: 0.8;
+}
+.advance-search-by-date .input-wrapper .mat-form-field .mat-form-field-wrapper .mat-form-field-flex .mat-form-field-infix {
+    border: 1px solid $gainsboro;
+    border-radius: 20px;
+    width: 100%;
+}
+.mat-field-border {
+    border: 1px solid $gainsboro !important;
+}
+.mat-grid-tile.select2-parent.select2-ledger .mat-grid-tile-content {
+    padding-left: 0px !important;
+}
+.ledg-head mat-grid-list mat-grid-tile.new-entry-date .mat-grid-tile-content {
+    padding-left: 10px !important;
+}
+.ledger-advance-search-field {
+    .mat-form-field-suffix {
+        right: 10px !important;
+    }
+    input.mat-input-element {
+        padding-left: 8px !important;
+        width: 96%;
+    }
+    input.mat-input-element.date-field, input.mat-input-element.description-field {
```

Consider nesting this to selector `input.mat-input-element` at line 2806
```suggestion
    &.date-field, &.description-field {
```